src/assets/_project/_blocks/components/forms/qg-address-autocomplete.js

```diff
@@ -0,0 +1,173 @@
+/*global qg, jQuery, google*/
+
+let qgInitAutocompleteAddress;
+
+(function (qg, $) {
+  'use strict';
+  let inputLocationId = 'qg-location-autocomplete';
+  let locationSelectionInProgress = true;
+
+  const el = {
+    $searchWidget: $('#qg-search-widget'),
+    $autoComplete: $('.qg-location-autocomplete'),
+    $latitude: $('#latitude'),
+    $longitude: $('#longitude'),
+    $form: $('#qg-search-widget-form'),
+  };
+
+  // getting and setting input fields value using query parameter
+  var setsValue = function () {
+    el.$form.find(':input:not(:checkbox):not(:radio), select, textarea').each(function () {
+      let name = $(this).attr('name');
+      let getParameterVal = qg.swe.getParameterByName($(this).attr('name'));
+      getParameterVal !== false ? $('[name="' + name + '"]').val(getParameterVal) : '';
+    }).end().find('input[type=checkbox], input[type=radio]').each(function () {
+      let name = $(this).attr('name');
+      let getParameterVal = qg.swe.getParameterByName(name);
+      getParameterVal !== false ? $('[value="' + getParameterVal + '"]').prop('checked', true) : '';
```

Comment:
Should this be prop('checked', false)?

Reply:
No , it should be true

Reply:
I meant
getParameterVal !== false ? $('[value="' + getParameterVal + '"]').prop('checked', true) : $('[value="' + getParameterVal + '"]').prop('checked', false);

If no value '' does nothing to the input. it should be unchecked.

Reply:
Radio button can only be checked at a time so if one is true others are false by default, no need to put in code.

Checkbox if checked place name so if paramter is there it means it is checked.


Place code and test if you like

<div class="form-group">
                    <div>
                      <h4>Radio Button Choice</h4>

                      <label for="radio-choice-1">Choice 1</label>
                      <input type="radio" name="radio-choice" id="radio-choice-1" tabindex="2" value="choice-1">

                      <label for="radio-choice-2">Choice 2</label>
                      <input type="radio" name="radio-choice" id="radio-choice-2" tabindex="3" value="choice-2">
                    </div>

                  </div>
                  <div class="form-group">
                    <input type="checkbox" name="vehicle1" value="Bike"> I have a bike<br>
                    <input type="checkbox" name="vehicle2" value="Car"> I have a car<br>
                    <input type="checkbox" name="vehicle3" value="Boat"> I have a boat<br>
                  </div>